core/validatorapi/validatorapi.go

```diff
@@ -1024,6 +1026,13 @@ func (c Component) ValidatorsByPubKey(ctx context.Context, stateID string, pubsh
 	return c.convertValidators(valMap)
 }
 
+// NodeVersion returns the current version of charon.
+func (Component) NodeVersion(context.Context) (string, error) {
+	commitSHA, _ := version.GitCommit()
+
+	return fmt.Sprintf("obolnetwork/charon/%s-%s/%s-%s", version.Version, commitSHA, runtime.GOARCH, runtime.GOOS), nil
```

Comment:
is `runtime.GOARCH` and `runtime.GOOS` present in the container?

Reply:
Yes it should be. I checked it running latest charon on my PC:
<img width="573" alt="Screenshot 2023-02-06 at 13 05 45" src="https://user-images.githubusercontent.com/43061995/216912119-f7ef018d-59de-4b3f-903a-7afdcc1b7afe.png">